packages/@aws-cdk/aws-appsync/lib/schema-base.ts

```diff
@@ -179,9 +218,28 @@ export class Directive {
   /**
    * the directive statement
    */
-  public readonly statement: string;
+  private statement: string;
```

Comment:
is changing the visibility here breaking?

Reply:
ah yes it is.. we use the toString() function instead to get the statement.. will add that to the commit body